packages/storage/src/bucket.rs

```diff
@@ -295,36 +281,6 @@ mod test {
         assert_eq!(loaded, expected);
     }
 
-    #[test]
-    fn update_mut_success() {
```

Comment:
It would be good to have this test (closure mutating a function local variable that can be read after the `update` call). Of course, using `update(birthday)` here not `update_mut(&mut birthday)`

Reply:
> (closure mutating a function local variable that can be read after the `update` call)

Ahhh, now I see the important part of this test: `old_age`. We need to remove the clutter that was already tested to make that visible. Will update